### PR TITLE
chore: bump gradle wrapper and disable gradle daemon

### LINKED ISSUE #2170: when building gradle plugin, gradle daemon remains running
The same issue than if flow https://github.com/vaadin/flow/issues/11159  but for the platform build

### PR DESCRIPTION
**Pull request by manolo**
fixes #2170

### CODE REVIEW

File: vaadin-gradle-plugin/gradlew.bat [hunk 1]
```diff
@@ -1,89 +1,89 @@
...
-rem Set variable GRADLE_EXIT_CONSOLE if you need the _script_ return code instead of
-rem the _cmd.exe /c_ return code!
-if  not "" == "%GRADLE_EXIT_CONSOLE%" exit 1
-exit /b 1
-
-:mainEnd
-if "%OS%"=="Windows_NT" endlocal
-
-:omega
+@rem
```
Comment (ZheSun88) [inline vaadin-gradle-plugin/gradlew.bat:1]: what happened with this file? all new?

**Review by ZheSun88 — APPROVED** [review]

File: vaadin-gradle-plugin/gradlew.bat (hunk 1 above)
Comment (ZheSun88) [reply vaadin-gradle-plugin/gradlew.bat:1]: tested locally, all work on windows.

**Comment by mshabarov** [comment]
`gradle-wrapper.jar` hasn't been updated, but it should to.

**Comment by manolo** [comment]
Yes, jar is updated sometimes, but not always that you run the upgrade command line order

**Pull request merged**